Sources/tuist/tuist.docc/Tutorials/Tuist-Cloud/Tuist-Cloud-Enterprise/Tuist Cloud - Enterprise Environment.tutorial

```diff
@@ -21,6 +21,7 @@
         
         | Environment variable | Description | Required | Default | Example |
         | --- | --- | --- | --- | --- |
+        | `TUIST_CLOUD_SELF_HOSTED` | This indicates to the base Docker image that this a self-hosted variant of Tuist Cloud. You should set this to `true` | Yes | No | `yes` |
```

Comment:
When designing how Tuist Cloud is configured for self-hosting, I'd aim at keeping things simple for the users and us be the ones absorbing the complexity. In the context of this line, I'd suggest to make self-hosted the default, and us be the ones setting that value to `false`. Or instead, have a variable that says that is hosted by us.